roles/generate_manifests/tasks/main.yml

```diff
@@ -38,6 +38,17 @@
     - agent-config.yaml.j2
     - install-config.yaml.j2
 
+# The agent-config files are eventually deleted after its usage, just
+# save them in a safe place in case of needing to check them
+- name: Save a backup of agent-config files
+  fetch:
+    src: "{{ item }}"
+    dest: "{{ repo_root_path }}"
```

Comment:
I think this should be `fetched_dest` for consistency with all the other fetched artefacts :)

Reply:
Sure! done. I'll recheck this because I think the automation is not working fine on this, I'll let you know when it's stable